### PR TITLE
[logging] better error logging for when opn fails to opn

### PR DESCRIPTION
**Pull request by gabrielcsapo**
Issue: #4342 

## What I did

- add better error logging message

screen shot 2018-10-10 at 6 04 38 pm

### CODE REVIEW

**Review by marvinhagemeister — APPROVED** [review]
Awesome, just checked out the branch locally and can confirm that this fixes the issue.

**Comment by gabrielcsapo** [comment]
@marvinhagemeister 🙌 woot thank you for checking it out!

**Review by igor-dv — APPROVED** [review]

File: lib/core/src/server/build-dev.js [hunk 1]
```diff
@@ -160,7 +160,13 @@ Would you like to run Storybook on port ${port} instead?`,
     if (program.smokeTest) {
       process.exit(stats.toJson().warnings.length ? 1 : 0);
     } else if (!program.ci) {
-      opn(address);
+      opn(address).catch(() => {
+        logger.error(`Could not open ${address} inside a browser. If you're running this command inside a
+docker container or on a CI, you need to pass the '--ci' flag to prevent opening a
+browser by default.`);
+
+        process.exit(1);
```
Comment (ndelangen) [inline lib/core/src/server/build-dev.js]: Why exit here?

Let's just continue to run, opening someone's browser isn't that important.

**Review by ndelangen — CHANGES_REQUESTED** [review]

**Comment by gabrielcsapo** [comment]
@ndelangen removed the process.exit

File: lib/core/src/server/build-dev.js [hunk 2]
```diff
@@ -117,7 +117,11 @@ export async function buildDevStandalone(options) {
     if (options.smokeTest) {
       process.exit(stats.toJson().warnings.length ? 1 : 0);
     } else if (!options.ci) {
-      opn(address);
+      opn(address).catch(() => {
+        logger.error(`Could not open ${address} inside a browser. If you're running this command inside a
+docker container or on a CI, you need to pass the '--ci' flag to prevent opening a
```
Comment (ndelangen) [inline lib/core/src/server/build-dev.js]: You can use `common-tags` and keep the whitespace easier to read.

Not a blocker for merger to me.

**Review by ndelangen — APPROVED** [review]

File: lib/core/src/server/build-dev.js (hunk 2 above)
Comment (gabrielcsapo) [reply lib/core/src/server/build-dev.js]: I'll add common-tags

**Comment by gabrielcsapo** [comment]
@ndelangen added common-tags as a dependency to lib/core and added the stripIndents code

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
merge when ready

**Pull request merged**